tests/unit/test_yamlparser.py

```diff
@@ -73,7 +73,7 @@ class DummyClass:
     assert type(a) == DummyClass
 
     with open(resource_filename('jina',
-                                '/'.join(('resources', 'executors.requests.%s.yml' % 'BaseExecutor')))) as fp:
+                                '/'.join(('resources', f'executors.requests.{"BaseExecutor"}.yml')))) as fp:
```

Comment:
I think here u can add directly `'executors.requests.BaseExecutor.yml'` right?

Reply:
Haha, strange that it was there